python/lsst/obs/hsc/makeHscRawVisitInfo.py

```diff
@@ -67,6 +64,13 @@ def setArgDict(self, md, argDict):
         argDict['era'] = self.eraFromLstAndLongitude(LST, self.observatory.getLongitude())
         argDict['darkTime'] = argDict['exposureTime']
 
+        # Rotation angle formula determined empirically from visual inspection
+        # of HSC images.  See DM-9111.
+        rotAngle = 270.0*degrees - self.popAngle(md, "INST-PA")
+        rotAngle.wrap()
```

Comment:
Ah, great!  I had actually originally tried to use `.wrap()` as you suggest, but I guess this was before the update since it failed at the time.  Change implemented.